.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.html"
+      - "**.css"
+      - "**.js"
+      - "**.md"
+      - "**.tsx"
+      - "**.json"
+  pull_request:
```

Comment:
https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#pull_request